src/bench.c

```diff
@@ -11,6 +11,31 @@
 #include "util.h"
 #include "bench.h"
 
+void help(int default_iters) {
+    printf("Benchmarks the following algorithms:\n");
+    printf("    1. ECDSA signing/verification\n");
+    printf("    2. ECDH key exchange (optional module)\n");
+    printf("    3. Public key recovery (optional module)\n");
+    printf("    4. Schnorr signatures (optional module)\n");
+    printf("\n");
+    printf("The default number of iterations for each benchmark is %d. This can be\n", default_iters);
+    printf("customized using the SECP256K1_BENCH_ITERS environment variable.\n");
+    printf("\n");
+    printf("Usage: ./bench [args]\n");
```

Comment:
Common practice is to use `argv[0]` for printing the binary name (which keeps working if it is renamed etc, or on systems where executable have extensions).

Reply:
I specifically asked for the hardcoded earlier in this PR: https://github.com/bitcoin-core/secp256k1/pull/1008#discussion_r745698996

Reply:
Ah, good point. I retract my comment.